transmart-core-db/grails-app/services/org/transmartproject/db/support/InQuery.groovy

```diff
@@ -25,12 +29,18 @@ import static org.transmartproject.db.support.DatabasePortabilityService.Databas
  */
 class InQuery {
 
-    public static final int MAX_LIST_SIZE = 1000
```

Comment:
1000 will also work for PostgeSQL, so why make this distinction?

Reply:
I thought it is better for the performance to have less 'OR' clauses, so it would be better to have larger 'IN', if possible for PostgreSQL. However, I haven't tested if it's working better, so if you know there is no big difference, I'll change it.